sphinx/sphinx.gradle

```diff
@@ -23,12 +23,22 @@ def copySphinxLib = tasks.register('copySphinxLib', Sync) {
 def sphinxDockerfile = tasks.register('sphinxDockerfile', Dockerfile) {
     destFile.set layout.buildDirectory.file('sphinx-image/Dockerfile')
     // Deephaven server python API requires that the wheel be installed to build, so we share this image
-    from 'deephaven/runtime-base:local-build'
+    from 'deephaven/server:local-build'
 
     copyFile "./wheel", "/wheel"
     copyFile "./lib", '${VIRTUAL_ENV}/lib/python3.7/site-packages/'
 
+    environmentVariable 'DEEPHAVEN_CLASSPATH', '/opt/deephaven/server-0.10.0/lib/*:/opt/deephaven/server-0.10.0/lib'
```

Comment:
We should template in `${project.version}`; and, I think we only need the `*` version.